scripts/Muon/GUI/Common/corrections_tab_widget/background_corrections_model.py

```diff
@@ -0,0 +1,166 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2021 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+from mantid.py36compat import dataclass
+
+from mantid.api import FunctionFactory, IFunction
+from Muon.GUI.Common.ADSHandler.ADS_calls import check_if_workspace_exist, retrieve_ws
+from Muon.GUI.Common.contexts.corrections_context import (CorrectionsContext, BACKGROUND_MODE_NONE, RUNS_ALL,
+                                                          GROUPS_ALL)
+from Muon.GUI.Common.contexts.muon_context import MuonContext
+from Muon.GUI.Common.corrections_tab_widget.corrections_model import CorrectionsModel
+
+DEFAULT_START_X = 5.0
+DEFAULT_END_X = 15.0
```

Comment:
the defaults should be the second half of the data (see the PSIBackgroundSubtraction algorithm). So for ISIS its about 15 to 30 and for PSI about 5 to 10.

Reply:
should these be in the context?